recipes/supervisor/post-link.sh

```diff
@@ -0,0 +1,6 @@
+if [[ $(id -u) == 0 ]]; then
+    mkdir -p /etc/supervisord/ && ln -s $PREFIX/etc/supervisord/ /etc/supervisord/
+    ln -s $PREFIX/etc/supervisord.conf /etc/supervisord.conf
```

Comment:
I see.  Yes, I think your scripts here are a good way to do what you want here, and making it work both on the user-level and root level is a good idea.

+1 to merge from me.